pkg/client/cmd/app.go

```diff
@@ -265,22 +265,34 @@ func appDel(cmd *cobra.Command, args []string) {
 	}
 
 	fmt.Print()
-	inputMsg := fmt.Sprintf(
-		"Are you sure you want to delete %s on %s? (yes/NO) ",
-		color.CyanString(name),
-		color.YellowString(currentClusterName),
-	)
-	s, _ := client.GetInput(inputMsg)
-	if s != "yes" {
-		fmt.Println("Delete process aborted!")
-		return
+
+	confirm, err := cmd.Flags().GetString("confirm")
+	if err != nil {
+		client.PrintErrorAndExit("Invalid app name")
```

Comment:
`Invalid confirm parameter`

Reply:
Also should it be a boolean?

Reply:
From what I understand, this `--confirm` flag will actually receive the app name. 

I'm not sure if passing the app name is a good approach. For example, if you use the `--force` flag on git, you're probably aware that the action you're about to do might be destructive.

Reply:
I think `--force` is much clearer (maybe `--silent`).